### DISCUSSION TITLE
6-0 Question 3 GeoServer Access Issues

### DISCUSSION THREAD

**Discussion by bryanlehnig** [ua-gist604b-f24/syllabus#24] [General]
Hello,

I am working on assignment 6-0. To answer question 3 we need to edit the symbol, upload the file, and in Edit Layer under the Publishing tab we need find the saved file and apply it. I can access the Edit Layer, but when I click on the Publishing tab I get an error. 
Access to redesigned-giggle-7vpwp57x9xx7cv5-8080.app.github.dev was denied
You don't have the user rights to view this page.

How do I get around this?

![Screenshot 2024-11-19 142856](https://github.com/user-attachments/assets/28c6983f-86ba-4200-b1fb-6c792721d36d)

**Comment by bryanlehnig** [comment]
Follow up:
It looks like for question five where it asks you to edit the order of the layers in spearfish and then save the edit I also get the same warning.

**Reply by jenw-ua** [reply]
I just posted an issue I'm having with Geoserver as well, but working toward question 3. Were you able to upload the sld file? Or even view anything after selecting the Publishing tab? I'm guessing you were able to, since you're on question 5? If so, what kind of magic did you do to make the file upload? Something so simple....yet not, when it doesn't work. Sorry I'm no help for question 5 yet.

**Reply by bryanlehnig** [reply]
Sorry to report that I did not fix anything. I got as far as I could with question 3 and moved on to the next instructions which don't require the completion of question 3. But the same error did occur for me in question 5. I was able to upload the sld file, but once I clicked the Publishing tab I got the error above. If you find a work around let me know haha

**Reply by jenw-ua** [reply]
Thanks, good to know. Will do.

**Comment by aaronbirr** [comment]
How do you have access to the styles and layer groups on geoserver? I only see layer preview. 
![Screenshot 2024-11-22 190342](https://github.com/user-attachments/assets/aef89ced-0cff-4648-a438-1cf059c1eac1)

**Reply by travisz09** [reply]
You have to log in using the username 'admin', password 'geoserver'.

**Reply by aaronbirr** [reply]
Thank you

**Comment by aldunn4** [comment]
I'm getting the same error and can't upload my sld. I've tried restarting my codespace and tried Chrome and Edge. None of the buttons appear to work on the New Style page.

It appears to work in Safari on my iPad. Apple discontinued Safari for Windows though so I'm putting my GeoServer URL and sld file in an email to myself, downloading it to the iPad and taking my screenshot for Q3 there.
